### ISSUE TITLE
MIAPPE

### ISSUE THREAD

**Issue by Enro** [rd-alliance/metadata-catalog-v2#9]
Thank you for your amazing catalog! Would it be possible to add MIAPPE to the catalog, if you confirm this standard for data in plant physiology falls within scope? https://www.miappe.org

Many thanks in advance

**Comment by alex-ball** [comment]
Yes, absolutely. The reason it isn't there already is historical with its roots in rd-alliance/metadata-directory#43.

That issue has been hanging around for far too long so yes, I would like to get it closed in time for the next RDA plenary and thereby kick off our content audit.

**Comment by cpommier** [comment]
Hi @alex-ball , this addition would indeed be useful. Is there anyway we can help ?
Thanks

**Comment by alex-ball** [comment]
@cpommier , Absolutely.

There is an introduction on [how to contribute](https://rdamsc.bath.ac.uk/contributing) records on the site itself.

If you want some examples of the sort of thing we're looking for, we have an entry for [MIxS](https://rdamsc.bath.ac.uk/msc/m108) (it could do with having its child schemes added as profiles) and I've recently put in basic entries for [CIMR](https://rdamsc.bath.ac.uk/msc/m130) and [GIATE](https://rdamsc.bath.ac.uk/msc/m131) to start to deal with this gap in our coverage.

Let me know if you run into any problems.
